README.md

```diff
@@ -1260,6 +1260,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Krishna Kumar](https://github.com/Krishnapro)
   - [Krishna R](https://github.com/Krishna-Ravi)
   - [Krishnansh Agarwal](https://github.com/krishnanshagarwal112)
+  - [Krishnal Panara](https://github.com/killshotroxs)
```

Comment:
To be in alphabetical order I believe your name should come after `Krishna R`? Given that `l` comes before `n` ( Krishna**l** and Krishna**n**sh)

Reply:
The above name wasn't there when i contributed...

Reply:
Yes it may have changed but this still means that your PR can only be merged if your name is in alphabetical order.  Please either update or if you are no longer interested in getting your PR merged then please close it.

Reply:
Done, i updated it.